### DISCUSSION TITLE
What are your show stopper bugs?

### DISCUSSION THREAD

**Discussion by nogginbox** [dotnet/maui#4305] [General]
I've seen lots of chatter around should I use Xamarin for new projects or is .NET MAUI ready to start working with.

If like me you've taken the plunge to use MAUI for your new project, what known bugs are stopping you from getting what you want done, or preventing you from demoing your app to early access users?

_(This question isn't meant to be a rant. If you use pre release open source software, you'd be mad to expect everything to work perfectly. The whole point is so that early users can find the issues. But there are a lot of issues and I thought this might be a way to highlight the ones people have found that they'd really like to be fixed so they can show off what they've been making. I'd also encourage people to comment on issues or leave emojis on ones that they'd like fixed to show which issues are affecting more people.)_

**Comment by nogginbox** [comment]
Mine are:

- [ ] **[5084. Triggers not working](/dotnet/maui/issues/5084)**  
- [x] [3877. NavigationPage.TitleView not showing](/dotnet/maui/issues/3877) - Fixed in Preview 14
- [x] [4267. On windows there is no Titlebar](/dotnet/maui/issues/4267) - Fixed in Preview 13
- [ ] [3486. Context Action Command not running on Windows App](/dotnet/maui/issues/3486)
- [ ] [3808. Placeholder text alignment is wrong on windows](/dotnet/maui/issues/3808)

**Reply by nogginbox** [reply]
Preview 13 has not fixed as many of these as I'd hoped, although I have noticed several other niggly UI things are working better.

Here's hoping for preview 14.

**Reply by jsuarezruiz** [reply]
The bug fixing effort will be increased shortly, right now the greatest effort is focused on completing APIs, etc.

**Reply by nogginbox** [reply]
Looking forward to that.

I am enjoying creating my MAUI app and I really want to show it to people to get feedback, but for now I'm going to wait till the bugs above are fixed.

**Comment by Dreamescaper** [comment]
Let's start with this one https://github.com/dotnet/maui/issues/3099 - I can't even install it.
(It worked for me before)

**Comment by danielftz** [comment]
Honestly the whole iOS development process has been real janky. From not being able to actually use VS Mac to many iOS unique visual bugs. I wish just "write once, run everywhere" were real.

**Comment by nogginbox** [comment]
I've found a new one that I really hope gets fixed soon. [Triggers aren't working](/dotnet/maui/issues/5084) so I can't style things based on the data being shown.
